test/privatekey.js

```diff
@@ -114,6 +140,22 @@ describe('PrivateKey', function() {
       }).to.throw('Invalid network');
     });
 
+    it('should not be able to instantiate private key WIF because of network mismatch', function() {
+      var invalidNet = {
+        name: 'invalidnet',
+        alias: 'invalidnet',
+        pubkeyhash: 0xFF,
+        privatekey: 0xFF,
+        scripthash: 0xFF,
+        xpubkey: 0xFFFFFFFF,
+        xprivkey: 0xFFFFFFFF,
+        networkMagic: 0xFFFFFFFF
+      };
+      expect(function(){
+        var a = new PrivateKey( wifNamecoin, invalidNet );
```

Comment:
`var a = new PrivateKey(wifNamecoin, 'testnet');` will also work here, since invalidNet isn't added to the networks.